release/gcb/run_daily_release.sh

```diff
@@ -16,6 +16,16 @@
 
 set -x
 
+# gsutil ls returns 0 if folder exists.
+gsutil ls "gs://$CB_GCS_FULL_STAGING_PATH"
+
+return_value=$?
+
+if [ $return_value = 0 ]; then
+    # Remove the existing folder.
+    gsutil -q rm -rf "gs://$CB_GCS_FULL_STAGING_PATH"
```

Comment:
you may as well just run it blindly. both gsutil ls and gsutil rm return 1 in case nothing is found.

You may also want to make sure it won't kill the script.
i.e. "set -o errexit" is not in the script chain.

alternatively, you can simply do

gsutil -q rm -rf "gs://$CB_GCS_FULL_STAGING_PATH" || echo "Staging path does not exist."

Reply:
Oh yeah, I like it.